app/helpers/pokemon_helper.rb

```diff
@@ -1,7 +1,41 @@
 # frozen_string_literal: true
 
-# pokemon
+# helper to upgrade pokemon view
 module PokemonHelper
+  include Pagy::Frontend
+
+  # added temporary until find another solution for page_nav method
+  def pagy_nav(pagy, pagy_id: nil, link_extra: "")
```

Comment:
Here I rewrite Pagy::Frontend method to make navigation look like I want. I'll find way how to avoid this in future.

Reply:
Consider using CSS to customize the look. What customization you have performed?